src/components/colorPicker/index.tsx

```diff
@@ -117,8 +117,9 @@ const ColorPicker = (props: Props) => {
 };
 
 ColorPicker.displayName = 'ColorPicker';
+ColorPicker.Dialog = ColorPickerDialog;
 
-export default asBaseComponent<Props>(ColorPicker);
+export default asBaseComponent<Props, {Dialog: typeof ColorPickerDialog}>(ColorPicker);
```

Comment:
Consider setting the static types like this (like we do in Button component)
```
ComponentStatics<typeof ColorPicker>
```

Reply:
Changed